CMakeLists.txt

```diff
@@ -72,7 +63,6 @@ ENDFUNCTION()
 
 LIST(APPEND CMAKE_MODULE_PATH cmake/Modules)
 
-IF(NOT KOKKOS_HAS_TRILINOS)
   set(CMAKE_DISABLE_SOURCE_CHANGES ON)
   set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
```

Comment:
Change `set` -> `SET` for consistency (drive-by).

Reply:
Let's try to make the CMake style uniform in a later pull request.